content/en/docs/instrumentation/java/manual.md

```diff
@@ -35,6 +35,87 @@ the most important piece of telemetry source-identifying info.
 
 For example:
 
+<details>
+  <summary>Maven top level dependencies. Click me.</summary>
+  Note that opentelemetry-semconv does not exist (yet) as 1.20.1 but only as 1.20.1-alpha.
+  
+  ```
+  
+  <project>
+      <dependencyManagement>
+          <dependencies>
+              <dependency>
+                  <groupId>io.opentelemetry</groupId>
+                  <artifactId>opentelemetry-bom</artifactId>
+                  <version>1.20.1</version>
```

Comment:
On other pages, we use variables to store the Java SDK version: https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/instrumentation/java/_index.md?plain=1#L10
There is an automatic process that updates these variables after each Java SDK/agent release.